tests/manage/mcg/test_verify_noobaa_status.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from ocs_ci.ocs import defaults
+from ocs_ci.utility.utils import run_async
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1, noobaa_cli_required
+)
+from ocs_ci.framework.testlib import polarion_id, bugzilla
+
+log = logging.getLogger(__name__)
+
+
+@tier1
```

Comment:
any reason to have this is tier1? I think it should be part of tier2

Reply:
Categorized as tier1 because this is doing basic status check.
Should I change it to tier2?

Reply:
yes please

Reply:
Done